### DISCUSSION TITLE
XML files in the nuget packages?

### DISCUSSION THREAD

**Discussion by wiverson** [supabase-community/supabase-csharp#68] [Ideas]
Disclaimer: coming from a Java background, been using Unity for several years now.

When using Maven packages, there's an option to include both source packages and javadoc API info. It looks like there's no real equivalent to bundling sources in .net, but the XML from comments can be included in an XML file. It looks like the nuget packages don't include those files. Is this something easy to add via a switch or something?

I can look into this if it's something that requires more info. @acupofjose is this easy or should I take a look...?

**Comment by acupofjose** [comment]
You just want the XML documentation file included? Yeah! Just a build flag

**Comment by wiverson** [comment]
I think so? The other nuget packages I'm using seem to include those, I think it's probably adding a flag to the right place in the release action? If you want me to dig it up and send a PR I can but if you know where to flip that works too.

**Reply by acupofjose** [reply]
v4.0.2 of gotrue-csharp includes the docs in its nuget release - can you check that out for me?

**Reply by wiverson** [reply]
https://nuget.info/packages/gotrue-csharp/4.0.2 - looks good to me! Add the same flag to the others?
